### DISCUSSION TITLE
Welcome to typst.ts Discussions!

### DISCUSSION THREAD

**Discussion by Myriad-Dreamin** [Myriad-Dreamin/typst.ts#358] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by justpankajdotcom** [comment]
Hi, Thanks for making this community. I am wondering if there's way to pass source code directly as a string to the compiler and then get the artifact, instead of reading it from a file? My use-case requires me to pass the source code from editor as a string directly to the compiler. Thank you so much for your efforts in building this amazing library.

**Comment by Myriad-Dreamin** [comment]
> Hi, Thanks for making this community. I am wondering if there's way to pass source code directly as a string to the compiler and then get the artifact, instead of reading it from a file? My use-case requires me to pass the source code from editor as a string directly to the compiler. Thank you so much for your efforts in building this amazing library.

@justpankajdotcom Please look at https://github.com/Myriad-Dreamin/typst.ts/issues/345#issuecomment-1722875419. I'm writing documentation for them :).

Btw, you can create a new discussion for specific problems. This would help a lot.
